frontend/src/pages/Client/Homepage/Homepage.js

```diff
@@ -13,6 +14,7 @@ const Homepage = () => {
       <PopularProductCard />
       <h1 className="section-h1">Our Reviews</h1>
       <ReviewPanel />
+      <FooterSection />
```

Comment:
This is a global footer so shouldn't be relegated to just being on the homepage. It should one scope up which would be the `UserRoutes`